tools/ci_build/github/azure-pipelines/post-merge-jobs.yml

```diff
@@ -1,143 +1,177 @@
+# these parameters are provided to selectively enable build configurations when running manually
+# all build configurations should be enabled by default
```

Comment:
👍

Reply:
You can put each of the jobs in to a separated stage. Then from the UI we can choose which jobs to run without the need of adding such parameters.